File: candidatures_web/app/Models/Compte.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Compte extends Model
{
    use HasFactory;
    
    protected $table = 'compte';
    protected $fillable = ['sexe','nom','prenom','email','date_naissance','mdp','mdp_crypted','nationalite','titre','adresse','adresse_comp','cp','ville','pays','numero','website'];

    public function candidatures()
    {
        return $this->hasMany(Candidature::class, 'compte', 'id');
    }

    public function cvs()
    {
        return $this->hasMany(CV::class, 'compte', 'id');
    }
}
?>

File: candidatures_ios/candidatures_ios/Compte.swift
//
//  Compte.swift
//  candidatures_ios
//
//  Created by Ashley Rakotoarisoa on 19/05/2026.
//
import Foundation

struct Compte: Codable {
    let id: Int
    let sexe: String?
    let nom: String?
    let prenom: String?
    let email: String?
    let date_naissance: String?
    let nationalite: String?
    let titre: String?
    let adresse: String?
    let ville: String?
    let pays: String?
    let numero: String?
    let website: String?
    let mdp: String?
}
